flask_resources/views/singleton.py

```diff
@@ -16,26 +16,52 @@ class SingletonView(BaseView):
     Note that the resource route should contain the `<id>` param.
     """
 
-    def __init__(self, resource=None, *args, **kwargs):
+    def __init__(self, *args, **kwargs):
         """Constructor."""
-        super(SingletonView, self).__init__(resource=resource, *args, **kwargs)
+        super(SingletonView, self).__init__(*args, **kwargs)
+        # is defined by response_handler decorator
+        self.response_handler = None
+
+    def resource_method(self):
```

Comment:
Let's make it a property to be consistent with the other implementations (https://github.com/inveniosoftware/flask-resources/blob/master/flask_resources/views/collections.py#L30)